src/sql/run/resultset.py

```diff
@@ -504,10 +505,11 @@ def _nonbreaking_spaces(match_obj):
 
 
 def _statement_is_select(statement):
-    statement_ = statement.lower().strip()
-    # duckdb also allows FROM without SELECT
-    return (
-        statement_.startswith("select")
-        or statement_.startswith("from")
-        or statement_.startswith("with")
-    )
+    if statement and isinstance(statement, str):
+        statement_ = statement.lower().strip()
+        # duckdb also allows FROM without SELECT
```

Comment:
looks like this comment is no longer necessary

Reply:
[Removed!](https://github.com/ploomber/jupysql/pull/797/commits/ff181bc0f6e274a0f1beb4f8018587fb3221ffd4#diff-ee061806ed7a4f5a79d8571f64241031f5c61240dbe59e548de0666a7996ab88L510)